cmd/ps.go

```diff
@@ -125,3 +146,35 @@ func printShipmentStatus(name string, shipment ComposeShipment) {
 
 	fmt.Println("-----")
 }
+
+//returns the primary port of a shipment
+func getShipmentPrimaryPort(dockerCompose project.APIProject, shipment ComposeShipment) (string, error) {
+
+	//get primary port of 1st service
+	for _, container := range shipment.Containers {
+		serviceConfig, success := dockerCompose.GetServiceConfig(container)
+		if !success {
+			log.Fatal("error getting service config")
+		}
+
+		parsedPort := strings.Split(serviceConfig.Ports[0], ":")
+		return parsedPort[0], nil
+	}
+	return "", errors.New("no shipment port found")
```

Comment:
or vice versa, let's surface all errors to the highest layer and handle the however we please. Would make it easier to test error cases out, but expecting errors to be returned vs just log.Fatal if something happens.